packages/http/src/index.ts

```diff
@@ -5,40 +5,17 @@ import mock from './mocker';
 import route from './router';
 import { validateInput, validateOutput } from './validator';
 export * from './types';
-import {
-  IHttpConfig,
-  IHttpMethod,
-  IHttpNameValue,
-  IHttpNameValues,
-  IHttpOperationConfig,
-  IHttpRequest,
-  IHttpResponse,
-  PickRequired,
-  PrismHttpComponents,
-  PrismHttpInstance,
-  ProblemJson,
-  ProblemJsonError,
-} from './types';
+export * from './getHttpOperations';
 
-const createInstance = (config: IHttpConfig, components?: PickRequired<Partial<PrismHttpComponents>, 'logger'>) => {
-  return factory<IHttpOperation, IHttpRequest, IHttpResponse, IHttpConfig>(
-    config,
+import { IHttpConfig, IHttpRequest, IHttpResponse, PickRequired, PrismHttpComponents } from './types';
+
+const createInstance = (
```

Comment:
It's so minor in comparison with the whole PR that I doubt I should comment ;)
```suggestion
export const createInstance = (
```